runtime/collections/lists.cpp

```diff
@@ -111,6 +120,10 @@ size_t hook_LIST_size_long(SortList list) {
   return list->size();
 }
 
+uint64_t hook_LIST_size64(SortList list) {
```

Comment:
hook_list_size_long already does this. Check for other similar duplicates that may exist.